.github/workflows/cluster_endtoend_vtgate_transaction_partial_keyspace.yml

```diff
@@ -0,0 +1,106 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING "make generate_ci_workflows"
+
+name: Cluster (vtgate_transaction) (partial keyspace)
+on: [push, pull_request]
+concurrency:
```

Comment:
Done. I also refactored test.go to allow specifying multiple config.*json. The config.json file was becoming pretty long and this will allow specifying tests in a modular way.